icons/perl/perl-original.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe you should optimize the SVG with the [SVG Viewer](https://www.svgviewer.dev/) (keep the `viewBox` and add a line at the end). Same with the `plain` version.

Reply:
Panquesito7 is right. Once this is optimized, I'll merge your icon @mattkoshela